osu.Game/Overlays/INamedOverlayComponent.cs

```diff
@@ -11,6 +11,12 @@ public interface INamedOverlayComponent
 
         LocalisableString Title { get; }
 
+        /// <summary>
+        /// An optional name used specifically for toolbar overlay toggle buttons.
+        /// If null, <see cref="Title"/> should be used directly instead.
+        /// </summary>
+        LocalisableString? ToolbarName => null;
+
         LocalisableString Description { get; }
```

Comment:
I'd say that this should also be named `ToolbarDescription` or similar, if we're going in this direction?

Reply:
I initially haven't done that as `Description` could potentially be used together with `Title` anywhere else, but given that the only usage of it is toolbar descriptions, I'll rename and make it optional.

Reply:
It seems this is the case already with settings overlay:
![CleanShot 2021-07-18 at 20 23 06](https://user-images.githubusercontent.com/22781491/126076474-9ee04b5f-5e23-486f-abda-5a4891e8b024.png)
![CleanShot 2021-07-18 at 20 23 25](https://user-images.githubusercontent.com/22781491/126076479-abf98c3a-533c-47c9-ae94-9522fe11ba5e.png)

I was initially going for `ShortName` rather than `ToolbarName`, does that sound better to avoid this sort of confusion?

Reply:
Honestly, I don't think it does.

I'd rather either unify the terminology to remove the need for this `ToolbarName` entirely, or have toolbar text/description on a separate interface. But I'll see what others have to say.

Reply:
I think it's okay for now. Quite an isolated usage and I'd rather it be explicit until otherwise required.